runtime/fake_dawn.py

```diff
@@ -43,21 +43,26 @@ def receiver(port, receive_queue):
         runtime_message = runtime_pb2.RuntimeData()
         runtime_message.ParseFromString(msg)
         receive_queue[0] = msg
-        print(runtime_message)
 
+def add_timestamps(msgqueue):
+    for i in range(10):
+        msg = notification_pb2.Notification()
+        msg.header = notification_pb2.Notification.TIMESTAMP_DOWN
+        msg.timestamps.append(time.perf_counter())
+        msg = msg.SerializeToString()
+        msgqueue.put(msg)
+    return msgqueue
 
-def tcp_relay(port):
+def tcp_relay(port, msgqueue=queue.Queue()):
```

Comment:
This default parameter seems like a possible source of bugs, as it's sharing the same queue among multiple calls to `tcp_relay`; is the optional parameter necessary?